### DISCUSSION TITLE
Show lap counter on OBCs and other suggestions

### DISCUSSION THREAD

**Discussion by peerstream-zz** [f1multiviewer/issue-tracker#243] [Feature requests]
Hello, a few suggestions:

1. Show lap counter on OBCs. If you are watching an OBC in full screen mode, it's useful to have a lap reference.
2. When replaying videos, change the Configure Sync with a one-click Sync Open Players button (at the moment it's a two click action that is not really necessary).
3. Consider changing the look of the best sector figures (for example, make them less bold or slightly smaller) just to differentiate the look between the current and best sector times
4. Add an option to hide the top bar of the live timing window. When arranging the layout with multiple windows, it's the only window that has the top bar displayed

Loving your app, thank you.

**Comment by jeppeolesen** [comment]
Hey @peerstream-zz 

If these are still relevant, please split them into separate feature requests - that makes them easier for us to handle.
